src/google_sheets_package/add_data_project_in_google_sheet.py

```diff
@@ -0,0 +1,68 @@
+import gspread
+from env import JSON_KEY_GOOGLE_API, URL_REPO
+import json
+from get_data_completed_project_package.project_data_serializer import ProjectDataSerializer
+from get_data_completed_project_package.get_all_data_from_repo_url import get_info_from_url
+from .table_fields_serializer import TableFieldsSerializer
+
+
+class AddDataProjectInGSheet:
+    def __init__(self):
+        """
+        Нужен для добавления проекта в таблицу итогов
+        """
+        # Подключение происходит с помощью ключа превращенного в dict
+        self.__dict_api_key_gsheet = json.loads(JSON_KEY_GOOGLE_API)
+        self.__google_sheet_object = gspread.service_account_from_dict(self.__dict_api_key_gsheet)
+        # Для получения индекса по имени листа, который используется для открытия нужного нам листа
+        self.__index_sheet_list = {
+            "hangman": 0,
+            "simulation": 1,
+            "currency-exchange": 2,
+            "tennis-scoreboard": 3,
+            "weather-viewer": 4,
+            "cloud-file-storage": 5,
+            "task-tracker": 6,
+            "другое": 7
+        }
+
+    def add_project_in_gsheet(self, project_data_object: ProjectDataSerializer, gsheets_name: str):
```

Comment:
Тут лучше подойдет `to`

```suggestion
    def add_project_to_gsheet(self, project_data_object: ProjectDataSerializer, gsheets_name: str):
```